testing/regress/ecl/genjoin.ecl

```diff
@@ -25,8 +25,10 @@ jrec := RECORD,MAXLENGTH(100)
     UNSIGNED1 i;
     STRING3 lstr;
     STRING3 rstr;
-    UNSIGNED1 c;
```

Comment:
oops... I actually saw that but since the file is checked in to github thought it must have been a historical remnant.  Should I change the script, or create separate versions for testing this new functionality?